test/collectors/shared_test.lua

```diff
@@ -16,3 +16,20 @@ g.test_different_order_in_label_pairs = function()
         {1, {a = 2, b = 1}},
     })
 end
+
+g.test_remove_metric_by_label = function()
+    local class = Shared:new_class('test_class', {'inc'})
+    local collector = class:new('test')
+    collector:inc(1, {a = 1, b = 2})
```

Comment:
Wait, is `{ string = number }` labels legal? Do export modules able to process it well?

It is not a question of this PR, maybe a ticket should be filed.

Reply:
Refer to https://github.com/tarantool/metrics/issues/289